azure-pipelines.yml

```diff
@@ -43,10 +43,29 @@ stages:
       buildOptions: 'USERNAME=admin SONIC_BUILD_JOBS=$(nproc) ${{ variables.VERSION_CONTROL_OPTIONS }}'
       jobGroups:
       - name: vs
+
+- stage: Build
+  pool: sonicbld
+  dependsOn: []
```

Comment:
Is it fixing something? do we need it?

Reply:
@qiluo-msft , it is to declare the build stage does not depend on any other stage.
A new stage BuildVS is added, the Test stage only depends on the BuildVS stage.
The Build stage, including broadcom, mellanox, marvell-armhf, can be run in parallel with BuildVS stage.
The Build stage will take more time than before, since marvel-armhf will take longer time to build.